asn1/CommonLedgerProtocol.asn

```diff
@@ -21,22 +21,24 @@ SideProtocolData ::= SEQUENCE OF SEQUENCE {
   protocolData OCTET STRING
 }
 
-Ack ::= SEQUENCE {
-  ack NULL,
+Request ::= SEQUENCE {
+  packet CHOICE {
```

Comment:
As commented in https://github.com/interledger/rfcs/pull/263#issuecomment-320779560, I think it's better to add an additional type than send an extra byte over the wire for every response.